packages/blocks/src/lib/navigation-map/components/scene.svelte

```diff
@@ -28,6 +28,8 @@ useMapLibreEvent('move', () => {
 useFrame(() => {
   renderer.clear();
 
+  axes.visible = scenes.length > 0
```

Comment:
[probably for another PR] To ensure this plays nicely with SvelteKit eventually, should these mutable global singletons be moved to a context?

Reply:
Yeah, this code is still making environment assumptions - and it should only ever run in the browser. Would like to pick your brain for following up since you've done some recent work with isomorphic sveltekit rendering